sched/Kconfig

```diff
@@ -1546,6 +1546,13 @@ endmenu # RTOS hooks
 
 menu "Signal Configuration"
 
+config SIG_PREALLOC_ACTIONS
+	int "Number of pre-allocated sigactions"
+	default 4 if DEFAULT_SMALL
```

Comment:
let's use 1 mean alloc/free directly, other value keeps previous behaviour:

1. allocate in batch
2. never return to heap